orchestra/static/orchestra/project/css/project.scss

```diff
@@ -0,0 +1,163 @@
+@import 'orchestra/common/css/common.scss';
+
+.project-management-modal {
+  font-size: 1.5rem;
+  padding: 15px;
+  .json-view {
+    margin-top: 10px;
+    max-height: 50vh;
+  }
+}
+
+.slack-modal {
+  @extend .project-management-modal;
+  input {
+    width: 200px;
+  }
+  .add-user {
+    cursor: pointer;
+    color: $light-green;
+  }
+  .remove-user {
+    cursor: pointer;
+    color: red;
+  }
+}
+
+.data-modal {
+  @extend .project-management-modal;
+  .header-wrapper {
+    overflow: auto;
+    .data-header {
+      float: left
+    }
+    .btn-admin {
+      float: right;
+    }
+  }
+}
+
+.revert-modal {
+  @extend .project-management-modal;
+  .header-wrapper {
+    overflow: auto;
+  }
+}
+
+.project-management {
+  margin-top: 75px;
+  font-size: 1.5rem;
+  position: relative;
+
+  .project-description {
+    font-size: 3rem;
+    margin: 0 10px;
+  }
+
+  .axis-wrapper {
+    position: relative;
+    display: inline-block;
+  }
+
+  .axis {
+    shape-rendering: crispEdges;
```

Comment:
Maybe because they're SVG-specific and that makes the CSS linter sad? I copied `shape-rendering` from a Bostock example and path is definitely supported so I don't think there's an issue.